jenkins/Jenkinsfile-publish

```diff
@@ -0,0 +1,48 @@
+// used by https://warpdrive-lab.dev.symphony.com/jenkins/view/DevX-Basilic/job/bdk-deploy-java/
+
+@Library(['sym-pipeline']) _
+
+import com.symphony.cicd.SymphonyCICDUtils
+
+cicdUtils = new SymphonyCICDUtils()
+
+
+node {
+
+    // default org and branch
+    def org = env.PARAM_REPO_OWNER ?: 'SymphonyPlatformSolutions'
+    def branch = env.PARAM_BUILD_BRANCH ?: 'master'
+
+    withCredentials([
+        file(credentialsId: 'JAVA_SDK_GPG_KEY', variable: 'ORG_GRADLE_PROJECT_KEY'),
+        string(credentialsId: 'JAVA_SDK_GPG_PASSPHRASE', variable: 'ORG_GRADLE_PROJECT_PASSPHRASE')
+    ]) {
+    
+      withEnv([
+        "PROJECT_TYPE=java",
+        "USE_OPENJDK11=false",
+        "DISABLE_SONAR=true",
+        "GIT_BRANCH=$PARAM_BUILD_BRANCH", 
+        "GIT_ORG=$PARAM_REPO_OWNER", 
+        "GIT_REPO=symphony-api-client-java"
+      ]) {
+    
+        stage('Git Checkout') {
+            gitCheckout()
+        }
+        
+        stage('Build project') {
+            sh './gradlew clean build'
+        }
+        
+        stage('Check jar file') {
+            sh 'ls -la ./symphony-bdk-legacy/symphony-api-client-java/target'
+            sh 'find . -type f -regex ".*/target/symphony-api-client-java-.*.jar" -exec echo {} \\;'
+        }
+        
+        stage('Deploy') {
+            sh './gradlew -PsymphonyRepoUrl=$PARAM_SYMPHONY_REPO -PsymphonyRepoUser=dev-services -PsymphonyRepoPassword=$TBD publish'
```

Comment:
TBD -> do you know how we can retrieve it?

Reply:
After checking, I saw we had credentials in jenkins